apso/src/main/scala/eu/shiftforward/apso/akka/http/ExtraMiscDirectives.scala

```diff
@@ -17,7 +18,9 @@ trait ExtraMiscDirectives {
   final val cacheControlNoCache: Directive0 =
     respondWithDefaultHeader(`Cache-Control`(`no-cache`, `no-store`, `must-revalidate`))
 
-  def cacheControlMaxAge(inMinutes: Option[Long]): Directive0 =
+  @deprecated("Use `eu.shiftforward.apso.akka.http.ExtraMiscDirectives.cacheControlMaxAge(Option[FiniteDuration])` " +
+    "instead", "2017/07/04")
+  def cacheControlMaxAge(inMinutes: Option[Long])(implicit d: DummyImplicit): Directive0 =
```

Comment:
This does not quite work as I expected, 'cause now I have to call this as `cacheControlMaxAge(client.browserRequestCacheDuration).apply{...}`...

:(